### ISSUE TITLE
I should still be able to contribute / donate to an event after its due date

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#6367]
In the same way that we can keep on adding expenses after an event happened, we should also be able to keep on donating to it (ie. some people pay for their ticket in cash at the event and anyone in the collective should be able to add that to the balance).

I've added a new tier to contribute, but it's disabled because (I guess) the event is over (https://opencollective.com/citizencorner/events/thunderosa-41ed2910)
Which brings another important point as a general UX rule: always give a reason for why something is disabled.

**Comment by Betree** [comment]
This is not supported for "regular" contributors at the moment. Like most event platforms, you can only buy a ticket before the event. There's a risk of having people buying a ticket without noticing the year if we change this. 

I'm not opposed to allowing contributions to tiers for everyone, as long as we keep it clear that the contribution is going to a past event.


Anyway, this should already be possible for admins as we keep on surfacing the tiers for them. @xdamman Could you confirm whether you're trying to contribute with your personal profile, as an admin?

![image](https://user-images.githubusercontent.com/1556356/214502603-9b181cd8-62b6-452a-b36b-17a17a7812b6.png)

![image](https://user-images.githubusercontent.com/1556356/214503021-9506a8b2-e2f1-499c-807b-db1274dc192b.png)

**Comment by xdamman** [comment]
<img width="1252" alt="Screenshot 2023-01-25 at 10 42 15 AM" src="https://user-images.githubusercontent.com/74358/214530114-5cc5abcf-0c38-447b-a1d3-a3980ae8a20c.png">

This is what I see (I'm logged in as an admin). No call to action. Tier disabled. 

This is what the tier looks like when I edit it:
<img width="1243" alt="Screenshot 2023-01-25 at 10 43 15 AM" src="https://user-images.githubusercontent.com/74358/214530304-121f5de8-774f-4c5b-9eb9-00f79a08f6a0.png">

That being said, I think anyone should be able to buy tickets (or contribute to any other tier) after the event took place, but you are right that it has to be clear (maybe even with a confirmation modal) that this is a past event. 

(likewise, uploading expenses after the event is not restricted to admins).

However, once the event is closed (archived) then of course, you can't contribute anymore or file any more expense.

**Comment by Betree** [comment]
> This is what I see (I'm logged in as an admin). No call to action. Tier disabled.

Thanks for sharing, the reason why I was not reproducing is that I was using "Enable flexible contributions", which does not seem impacted by this bug (a possible workaround if you want to do it that way). I'm pushing a fix for that in https://github.com/opencollective/opencollective-frontend/pull/8582.

> likewise, uploading expenses after the event is not restricted to admins

I don't think that the comparison is meaningful, the use cases are really different:
It's normal to get reimbursed for things after an event is over, many expenses will occur on the same day (food, travel, etc) or will have to be paid afterward (renting the space, covering the costs for speakers traveling back, etc).
On the other hand, you're supposed to buy tickets in advance or on-site (which is why we allow contributions up to 24h after the exact "ends at"). I've personally never attended an event with the ability to buy my tickets after participating.

I understand that it might be handy is some situations, but it's an edge case that some large platforms such as EventBrite are not supporting and I would be concerned about introducing the opposite problem of people contributing to past events thinking they'll happen in the future if the UI is not crystal clear.

I'll let @opencollective/design look at that part.

**Comment by filipesmedeiros** [comment]
> I've personally never attended an event with the ability to buy my tickets after participating.

In our case, it's not about allowing people to buy tickets after the fact, but actually _donating_ to a specific event they went to, kind of like having a tip jar at the exit door. That's why OpenCollective has "Tickets" and "Contributions" separately, no?

**Comment by Betree** [comment]
@filipesmedeiros Yes, you can still create donation tiers on your event, and those will stay even after the event is over.

**Comment by filipesmedeiros** [comment]
@Betree huh… not the default one, it seems!

EDIT: I don't think any donation tier stays on after the event is over: https://opencollective.com/decrescimento-pt/events/exibicao-do-filme-the-cost-of-growth-ab132a7f (this event has a donation tier created, in addition to the default tier)

**Comment by Betree** [comment]
@filipesmedeiros I'm sorry about the misleading instructions, you're absolutely right. The last changes on these are from three years ago and I was not remembering them correctly. Events donation tiers do disappear when the event is over.
This is enforced by backend logic [here](https://github.com/opencollective/opencollective-api/blob/d48b935b39197e68ff38567bacf49514f50a4866/server/graphql/common/features.ts#L51) and frontend logic [here](https://github.com/opencollective/opencollective-frontend/blob/159c82b604a5f267500c828e72c44c1f57b2727f/components/collective-page/sections/Contribute.js#L176). 

Your use case is right, and I agree with keeping donation tiers on the events even after it's over. Users can easily archive the event if they don't want to receive any kind of contributions on it anymore.

To address this, we'll need to update the two paths mentioned above and add a some instructions in the dashboard tiers section for events to explain that behavior.
